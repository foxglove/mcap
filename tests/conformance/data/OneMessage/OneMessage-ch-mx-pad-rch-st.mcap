version https://git-lfs.github.com/spec/v1
oid sha256:35a616800aaca45437a8da578db87d14970e1044aa56c127d945be4b014de352
size 397
