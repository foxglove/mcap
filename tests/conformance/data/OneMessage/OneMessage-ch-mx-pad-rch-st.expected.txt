Header library=s:"" profile=s:""
Schema data=b:<040506> encoding=s:c id=i:1 name=s:Example
ChannelInfo id=i:1 message_encoding=s:a metadata=m:{s:foo=s:bar} schema_id=i:1 topic=s:example
Message channel_id=i:1 data=b:<010203> log_time=n:2 publish_time=n:1 sequence=i:10
ChannelInfo id=i:1 message_encoding=s:a metadata=m:{s:foo=s:bar} schema_id=i:1 topic=s:example
Statistics attachment_count=i:0 channel_count=i:1 channel_message_counts=m:{i:1=n:1} chunk_count=i:1 message_count=n:1
ChunkIndex chunk_length=n:162 chunk_start_offset=n:28 compressed_size=n:115 compression=s:"" end_time=n:2 message_index_length=n:34 message_index_offsets=m:{i:1=n:192} start_time=n:2 uncompressed_size=n:115
Footer summary_crc=i:0 summary_offset_start=n:0 summary_start=n:242
